File: DTO/Response/UserResponse.cs
namespace BerryNameApi.DTO.Response
{
    public class UserResponse
    {
        public required Guid Id { get; set; }
        public required string Name { get; set; }
        public required DateTime CreatedAt { get; set; }
        public required DateTime UpdatedAt { get; set; }
    }
}

File: src/dto/res/user_response.rs
use chrono::{DateTime, Utc};
use serde::{Deserialize, Serialize};
use uuid::Uuid;

use crate::dto::res::company_response::{deserialize_uuid, serialize_uuid};

#[derive(Debug, Serialize, Deserialize, Clone)]
pub struct UserResponse {
    #[serde(serialize_with = "serialize_uuid", deserialize_with = "deserialize_uuid")]
    pub id: Uuid,
    pub email: String,
    pub name: String,
    pub role: String,
    pub created_at: DateTime<Utc>,
    pub updated_at: DateTime<Utc>,
}
